### ISSUE TITLE
Four tests ignored on pytest 5

### ISSUE THREAD

**Issue by jayvdb** [google/sre_yield#15]
Output in pytest 5.2.4
```py
sre_yield/tests/test_bigrange.py:34
  sre_yield/tests/test_bigrange.py:34: PytestCollectionWarning: yield tests were removed in pytest 4.0 - test_all will be ignored
    def test_all():

sre_yield/tests/test_fastdivmod.py:139
  sre_yield/tests/test_fastdivmod.py:139: PytestCollectionWarning: yield tests were removed in pytest 4.0 - test_correctness_big_numbers will be ignored
    def test_correctness_big_numbers():

sre_yield/tests/test_fastdivmod.py:168
  sre_yield/tests/test_fastdivmod.py:168: PytestCollectionWarning: yield tests were removed in pytest 4.0 - test_powersum will be ignored
    def test_powersum():

sre_yield/tests/test_slicing.py:52
  sre_yield/tests/test_slicing.py:52: PytestCollectionWarning: yield tests were removed in pytest 4.0 - test_parity will be ignored
    def test_parity():
```

**Comment by thatch** [comment]
Those work on nose but nothing else I suppose. I have a draft to use data_provider instead, it's basically parameterized.expand and appears to work fine on pytest as well as standard unittest. Will push in a couple hours.

**Comment by jayvdb** [comment]
I've been using https://pypi.org/project/unittest_expander/ for when I want standard unittest compatibility.  Looking forward to seeing what `data_provider` .. maybe https://pypi.org/project/unittest-data-provider/ ?  I did set that up at https://build.opensuse.org/package/show/home:jayvdb:py-new/python-unittest-data-provider , but never got around to testing it.

**Comment by thatch** [comment]
Fixed by #20
